quesma/quesma/router.go

```diff
@@ -200,33 +200,38 @@ func ConfigureRouter(cfg *config.QuesmaConfiguration, sr schema.Registry, lm *cl
 		return elasticsearchQueryResult(string(responseBody), http.StatusOK), nil
 	})
 
-	router.Register(routes.IndexMappingPath, and(method("PUT"), matchedAgainstPattern(tableResolver)), func(ctx context.Context, req *mux.Request) (*mux.Result, error) {
-		index := req.Params["index"]
+	router.Register(routes.IndexMappingPath, and(method("GET", "PUT"), matchedAgainstPattern(tableResolver)), func(ctx context.Context, req *mux.Request) (*mux.Result, error) {
```

Comment:
We should be really vocal about ConfigureRouter's behavior w.r.t. handler registering and matching, please add a comment about its behavior (preferably above the function signature so that's its visible in IDE)